mimic.py

```diff
@@ -61,8 +74,15 @@ def print_mimic(mimic_dict, word):
         - Randomly select a new seed word from this word list
         - Repeat this process 200 times
     """
-    # +++your code here+++
-    raise NotImplementedError("Get to Work!")
+    if not word:
+        word = random.choice(mimic_dict.keys())
+    output = [word]
+
+    for story in xrange(0, 199):
+        word = random.choice(mimic_dict[word])
+        output.append(word)
+
```

Comment:
You probably need to process the output and print it out. There should be an outcome on your terminal after you run it.